.gitlab-ci.yml

```diff
@@ -109,7 +109,7 @@ publish:docker:release:
         || ( echo "no docker credentials provided"; exit 1 )
     - docker login -u "$Docker_Hub_User_Parity" -p "$Docker_Hub_Pass_Parity"
     - docker info
-    - VERSION="$(./target/release/substrate --version | sed -n -r 's/^substrate ([0-9.]+-.*-[0-9a-f]{8})-.*$/\1/p')"
+    - VERSION="$(./target/release/substrate --version | sed -n -r 's/^substrate ([0-9.]+.*-[0-9a-f]{7,8})-.*$/\1/p')"
```

Comment:
Why `{7,8}`?

Reply:
good question :thinking: which I cannot answer.
I found this `substrate 1.0.0-beta-3d549e9e-x86_64-linux-gnu` and that `substrate 0.1.0-1ba9688-x86_64-linux-gnu` in the wild.